src/Actions/Family/ListFamilies.php

```diff
@@ -88,6 +88,8 @@ public function __invoke(Request $request, Response $response, array $vars): Res
             '/families'
         );
 
+        $this->entityManager->getFilters()->disable('family');
```

Comment:
I don't believe this is necessary.

Reply:
hahaha, I typed that to show you the syntax, whooooops :V